packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -245,6 +251,26 @@ export class FilterGroup {
     return this.addHeadBranchFilter(branchName, false);
   }
 
+  /**
+   * Create a new FilterGroup with an added condition:
+   * the event must affect a head commit with the given message.
+   *
+   * @param commitMessage the commit message (can be a regular expression)
+   */
+  public andCommitMessageIs(commitMessage: string): FilterGroup {
+    return this.addCommitMessageFilter(commitMessage, true);
+  }
+
+  /**
+   * Create a new FilterGroup with an added condition:
+   * the event must not affect a head commit with the given message.
+   *
+   * @param commitMessage the commit message (can be a regular expression)
+   */
+  public andCommitMessageIsNot(commitMessage: string): FilterGroup {
+    return this.addCommitMessageFilter(commitMessage, true);
```

Comment:
I believe this should be `false` here, right...?

```suggestion
    return this.addCommitMessageFilter(commitMessage, false);
```

Reply:
:sweat_smile: haha yes, good catch